.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,34 @@
+---
+name: Feature request
+about: Suggest a new feature in rust-libp2p.
+---
+
+<!--
+Note: If you'd like to suggest an feature related to libp2p but not specifically related to the rust implementation, please file an issue at https://github.com/libp2p/libp2p instead. Even better, create a new topic on the forums (https://discuss.libp2p.io).
```

Comment:
```suggestion
Note: If you'd like to suggest an feature related to libp2p but not specifically related to the rust implementation, please file an issue at https://github.com/libp2p/specs instead.
```

Is anyone still using `libp2p/libp2p`?

I would suggest we keep discuss.libp2p.io for helping folks. I would prefer any design work to happen on libp2p/specs.

Reply:
I copied that part from the go-libp2p template: https://github.com/libp2p/go-libp2p/blob/e8d3df92f7a6f72475dcd028496be2c6923fcb3a/.github/ISSUE_TEMPLATE/feature.md?plain=1#L8 (cc @marten-seemann).
But keeping design work to a single place, that is libp2p/specs, sounds good to me.